### DISCUSSION TITLE
listing available radio stations

### DISCUSSION THREAD

**Discussion by aristosv** [AzuraCast/AzuraCast#6340] [General]
I set up azuracast and configured 5 radio stations. Is there a way to display these stations? Like a page that azuracast generates, which lists all the available radio stations? Thanks

**Comment by BusterNeece** [comment]
@aristosv If all the stations are public, you can go to `/api/nowplaying` and see them all as a JSON feed.

**Comment by aristosv** [comment]
I'm assuming I can somehow use this to generate a static website. It would be nice to have something directly from Azuracast, showcasing the stations it's hosting. Thanks for the reply though.

**Comment by rogercreagh** [comment]
It seems that you can also go to `/api/stations` which will give just a list of the stations on the server  without the now playing and history information. Makes it a little easier to pull out what you want.

**Comment by rogercreagh** [comment]
you can also use `/api/stations` which will just give you the static station info without the now-playing and history, which makes it slightly easier.

If you go there in a browser you may find a "pretty-print" or similar button or checkbox at the top which formats the string of text so you can find the bits you want easily.

For a static website you are presumably going to hard-code the server url to get the info anyway, so you may as well just manually hard code the station names and links you get from `api/stations`.
